src/components/Action/Action.vue

```diff
@@ -100,16 +124,41 @@ export default {
 	methods: {
 		toggleMenu() {
 			this.opened = !this.opened
+			// focus first on menu open after opening the menu
+			if (this.opened) {
+				this.$nextTick(function() {
+					this.focusFirstAction()
+				})
+			}
 			this.$emit('update:open', this.opened)
 		},
 		closeMenu() {
 			this.opened = false
 			this.$emit('update:open', this.opened)
 		},
-		mainActionElement() {
-			return {
-				is: this.isSingleAction ? 'a' : 'div'
-			}
+		focusAction() {
+			this.$el.querySelectorAll('.focusable')[this.focusIndex].focus()
+		},
+		focusPreviousAction() {
+			this.focusIndex = Math.max(this.focusIndex - 1, 0)
+			this.focusAction()
+		},
+		focusNextAction() {
+			this.focusIndex = Math.min(this.focusIndex + 1, this.$el.querySelectorAll('.focusable').length - 1)
+			this.focusAction()
+		},
+		focusFirstAction() {
+			this.focusIndex = 0
+			this.focusAction()
+		},
+		focusLastAction() {
+			this.focusIndex = this.$el.querySelectorAll('.focusable').length - 1
+			this.focusAction()
+		},
+		// exec the first action and prevent default
+		execFirstAction(e) {
```

Comment:
Strangely using `click.prevent` does not work. So I had to do that :woman_shrugging: